src/audits/UnsupportedAriaAttribute.js

```diff
@@ -49,7 +51,15 @@ goog.require('axs.utils');
             }
             else {
                 // This test ignores the fact that some HTML elements should not take even global attributes.
-                supported = axs.constants.GLOBAL_PROPERTIES;
+                if(globalProperties) {  // if we have previously computed this value, use it
```

Comment:
This is nice, but I think it's an unnecessary optimisation. Other initialisation happens right inside Constants.js, e.g. https://github.com/GoogleChrome/accessibility-developer-tools/blob/master/src/js/Constants.js#L982 - could you use this pattern instead?